### DISCUSSION TITLE
Classification based RNA-Seq Script editing

### DISCUSSION THREAD

**Discussion by Groshack** [biostars/biostar-handbook-issues#212] [Q&A]
Hi everyone, 

I have completed the RNA-Seq by example and applied it to my samples which has been very successful.  I have found Kallisto and EdgeR to be the most suitable common ground for my application. 

I have been asked to re-run the Kallisto EdgeR analysis but to do a glmLRT and glmQLFTest instead of the exactTest. Can the "edger.r" script be edited to accomplish this and if so, how? 

Cheers,
Groshack

**Comment by ialbert** [comment]
I think here you would need to search Google and you will hit a number of posts on the subject. I do think just a few lines would need to be edited, as the returned objects should match. But one would need to try to see.

Another possibility would be to find someone else's script, then add to that some of the solutions from the edge.r implementation.

I do plan on an expansion that talks about statistics and the various choices, but would be bit on a longer term.

**Reply by Groshack** [reply]
That was my first thought also. I think I am most of the way to the solution but seem to be struggling with the design variable. I will post my solution if I manage to crack it. Admittedly, I started scripting a couple of months ago. The leading curve is jarring and I am a lot slower than I thought I would be.

**Reply by ericjuo** [reply]
@Groshack I were struggling with design matrix as well, and I found this YouTube video very helpful to understand what is general linear model and design matrix. https://www.youtube.com/watch?v=AVxojKRPsWg&list=PLIeGtxpvyG-KwtuGE4kKONQHILQiL31qZ
